.test-infra/jenkins/job_LoadTests_GBK_Python.groovy

```diff
@@ -22,119 +22,138 @@ import InfluxDBCredentialsHelper
       test           : 'apache_beam.testing.load_tests.group_by_key_test',
       runner         : CommonTestProperties.Runner.DATAFLOW,
       pipelineOptions: [
-        job_name             : 'load-tests-python-dataflow-batch-gbk-4-' + now,
+        job_name             : "load-tests-python-dataflow-${mode}-gbk-4-${now}",
         project              : 'apache-beam-testing',
         region               : 'us-central1',
         temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
         publish_to_big_query : true,
         metrics_dataset      : datasetName,
-        metrics_table        : 'python_dataflow_batch_gbk_4',
-        influx_measurement   : 'python_batch_gbk_4',
+        metrics_table        : "python_dataflow_${mode}_gbk_4",
+        influx_measurement   : "python_${mode}_gbk_4",
         input_options        : '\'{"num_records": 5000000,' +
         '"key_size": 10,' +
         '"value_size": 90}\'',
         iterations           : 1,
         fanout               : 4,
-        num_workers          : 5,
-        autoscaling_algorithm: "NONE"
+        num_workers          : 16,
+        autoscaling_algorithm: 'NONE',
       ]
     ],
     [
       title          : 'GroupByKey Python Load test: fanout 8 times with 2GB 10-byte records total',
       test           : 'apache_beam.testing.load_tests.group_by_key_test',
       runner         : CommonTestProperties.Runner.DATAFLOW,
       pipelineOptions: [
-        job_name             : 'load-tests-python-dataflow-batch-gbk-5-' + now,
+        job_name             : "load-tests-python-dataflow-${mode}-gbk-5-${now}",
         project              : 'apache-beam-testing',
         region               : 'us-central1',
         temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
         publish_to_big_query : true,
         metrics_dataset      : datasetName,
-        metrics_table        : 'python_dataflow_batch_gbk_5',
-        influx_measurement   : 'python_batch_gbk_5',
+        metrics_table        : "python_dataflow_${mode}_gbk_5",
+        influx_measurement   : "python_${mode}_gbk_5",
         input_options        : '\'{"num_records": 2500000,' +
         '"key_size": 10,' +
         '"value_size": 90}\'',
         iterations           : 1,
         fanout               : 8,
-        num_workers          : 5,
-        autoscaling_algorithm: "NONE"
+        num_workers          : 16,
+        autoscaling_algorithm: 'NONE',
       ]
     ],
-  ].each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  ]
+  .each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+  .each { test -> (mode != 'streaming') ?: addStreamingOptions(test) }
+  .withIndex().collectMany { test, i ->
+    mode == 'streaming' && STREAMING_TESTS_TO_SKIP.contains(i + 1) ? []: [test]
+  }
+}
+
+def addStreamingOptions(test) {
+  test.pipelineOptions << [streaming: null, experiments: 'use_runner_v2',
+    enable_streaming_engine: null ]
+}
```

Comment:
I removed `--enable_streaming_engine`, since it is now being added automatically when using `use_runner_v2`:  https://github.com/apache/beam/pull/12585